File: A1/A1-013/A1-013.py
# --------------------------------------------------
# File Name : A1-013.py
# Problem   : Safe Password
# Author    : Worralop Srichainont
# Date      : 2026-02-08
# --------------------------------------------------

# Input
letter = input().strip()
num = input().strip()

# Output
if letter == "H" and num == "4567":
    print("safe unlocked")
elif letter == "H":
    print("safe locked - change digit")
elif num == "4567":
    print("safe locked - change char")
else:
    print("safe locked")

File: A1/A1-013/A1-013.cpp
#include<iostream>
using namespace std;

int main()
{
    // This makes std::cin and std::cout faster
    ios_base::sync_with_stdio(false); cin.tie(0);

    // Input
    string letter, num;
    cin >> letter >> num;
    // Output
    if(letter == "H" && num == "4567") { cout << "safe unlocked"; }
    else if(letter == "H") { cout << "safe locked - change digit"; }
    else if(num == "4567") { cout << "safe locked - change char"; }
    else { cout << "safe locked"; }
    return 0;
}